src/Components/Layout/WorkshopEditor.tsx

```diff
@@ -10,10 +10,23 @@ import draftToHtml from "draftjs-to-html";
 import "react-draft-wysiwyg/dist/react-draft-wysiwyg.css";
 import { Button } from "@material-ui/core";
 
-class WorkshopEditor extends React.Component {
-  constructor(props) {
+interface WorkshopEditorProps {
+  closeWindow(newText: string): void;
+  content: any;
+}
+
+interface WorkshopEditorState {
+  editorState: any;
```

Comment:
VScode should give an inferred type for these two variables, recommend using them

Reply:
I believe theres a editor state type definition if you look for draft js types.